source/scss/components/_loaders.scss

```diff
@@ -0,0 +1,132 @@
+$radius: 15px;
+$one-third: 7.5px;
+$two-thirds: 13px;
+$animation: ease-in-out;
+$animation-period: 1.44s;
+$line-animation-period: -.12s;
+
+.loader {
+  align-items: center;
+  box-sizing: border-box;
+  display: flex;
+  flex: 0 1 auto;
+  flex-direction: column;
+  height: 46px;
+  justify-content: center;
+}
+
+.loading-spinner {
```

Comment:
As you already have properties for `.loading-spinner` you can nest all the child elements inside :)